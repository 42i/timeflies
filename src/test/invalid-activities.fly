wp hotdog

day 2012-10-25 8 14
- hotdog 4.5
- hotcat 1
- blueant three

day 2012-10-26 8 14
- hotdog oneandahalf
