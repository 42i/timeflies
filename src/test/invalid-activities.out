invalid-activities.fly:5: ERROR : invalid activity work package "hotcat".
invalid-activities.fly:6: ERROR : invalid activity work package "blueant".
invalid-activities.fly:6: ERROR : invalid activity duration "three".
invalid-activities.fly:9: ERROR : invalid activity duration "oneandahalf".
